hash/snes.xml

```diff
@@ -37792,6 +37892,42 @@ List of unclassified roms
 		</part>
 	</software>
 
+	<software name="claskong">
+		<description>Classic Kong Complete (v2.01)</description>
+		<year>2012</year>
+		<publisher>Piko Interactive</publisher>
+		<part name="cart" interface="snes_cart">
+			<feature name="slot" value="lorom" />
+			<dataarea name="rom" size="262144">
+				<rom name="Classic Kong Complete (World) (v2.01) (Aftermarket) (Unl).sfc" size="262144" crc="8fa368be" sha1="33879e1a173b6f3a9e3deb160f6def1ddeb38463" offset="0x000000" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="claskong20" cloneof="claskong">
+		<description>Classic Kong Complete (v2.0)</description>
+		<year>2012</year>
+		<publisher>Piko Interactive</publisher>
+		<part name="cart" interface="snes_cart">
+			<feature name="slot" value="lorom" />
+			<dataarea name="rom" size="262144">
+				<rom name="Classic Kong Complete (World) (v2.0) (Aftermarket) (Unl).sfc" size="262144" crc="bdc32ad8" sha1="abac92ccc09e4e21b45da3104bbe3d28b6c0bb24" offset="0x000000" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="classkng" cloneof="claskong">
+		<description>Classic Kong</description>
+		<year>2012</year>
+		<publisher>Piko Interactive</publisher>
+		<part name="cart" interface="snes_cart">
+			<feature name="slot" value="lorom" />
+			<dataarea name="rom" size="262144">
+				<rom name="Classic Kong (World) (Aftermarket) (Unl).sfc" size="262144" crc="b068407d" sha1="241b3247039d3e5e269a0f878d0021419a2a9d3d" offset="0x000000" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
I don’t like the short name for the oldest version – add a suffix to the parent’s short name (e.g. `claskongo` for oldest/original or something).